### PR TITLE
Frequently Asked Questions

### PR DESCRIPTION
**Pull request by jonthegeek**
This will almost certainly get more updates after we announce the name change, but this will work as a start. Let me know if you see anything to fix before we announce!

### CODE REVIEW

File: faq.qmd [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+But we host [book clubs](bookclubs.qmd) for many other R book (such as [*Advanced R*](https://r4ds.io/advr), [*Mastering Shiny*](https://r4ds.io/mshiny), and [*Tidy Modeling with R*](https://r4ds.io/tmwr)), and even other programming languages (such as [*Python for Data Analysis*](https://r4ds.io/py4da) and [*Fundamentals of Numerical Computation with Julia*](https://r4ds.io/fnc)).
+
+### Question: Why did you change your name? {#name-change}
+
+Answer: We wanted a name that caused less confusion.
+
+We launched in 2017 as an online space for learners and mentors to gather to read [*R for Data Science*](https://r4ds.hadley.nz/) (R4DS) by Hadley Wickham and Garrett Grolemund together. 
+The name "R4DS Online Learning Community" made sense for that group. 
+
+However, [we now support much more than just that one book](#r4ds-club).
```
Comment (tanho63) [inline faq.qmd]: This should be the first one I think

Comment (jonthegeek) [reply faq.qmd]: Good call, I was just getting them down, didn't do any sorting. Will update with order...

File: faq.qmd [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Frequently Asked Questions"
+format:
+  html:
+    toc: true
+---
+
+### Question: Are you part of (Posit, Google, etc)? {#independent}
+
+Answer: No!
```
Comment (tanho63) [inline faq.qmd]: may consider https://quarto.org/docs/authoring/callouts.html for extra emphasis later on? or formatting as collapsible callouts? not important for v1

Comment (jonthegeek) [reply faq.qmd]: Yeah, I figure there's probably a FAQ plugin or something somewhere. For now I went with "good enough".

File: faq.qmd [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+We have received donations from [Posit](https://posit.co/), and Google mentions us in their [Data Analytics Certificate](https://grow.google/certificates/data-analytics) courses, but we are not part of either organization.
+
+### Question: Are you changing your focus from R to Python? {#focus}
+
+Answer: No! 
+We seek to support *all* data science learners.
+
+We began with R, and our internal tools are written in R.
+However, we know that many data scientists use Python, Julia, and other languages in their day-to-day work.
+We want to help data science learners learn all of the skills they need, regardless of whether those skills involve R, Python, Julia, JavaScript, or no programming language at all.
```
Comment (tanho63) [inline faq.qmd]:
```suggestion
We began with R, our internal tools are written in R, and many of our users, book clubs and channels focus on R. 
However, we know that data scientists use many languages in their day-to-day-work, including Python, Julia, SQL.
We want to help data science learners learn all of the skills they need, regardless of whether those skills involve R, Python, Julia, SQL, JavaScript, or no programming language at all.
```

Comment (jonthegeek) [reply faq.qmd]: Fixed in separate commit. Note that "day-to-day" is an adjective phrase modifying "work", thus "day-to-day work".

**Pull request merged**